frontend/components/Main/MainOlasBalance.tsx

```diff
@@ -1,16 +1,103 @@
-import { Skeleton } from 'antd';
+import { InfoCircleOutlined } from '@ant-design/icons';
+import { Flex, Skeleton, Tooltip, Typography } from 'antd';
 import { useMemo } from 'react';
 import styled from 'styled-components';
 
 import { balanceFormat } from '@/common-util/numberFormatters';
 import { UNICODE_SYMBOLS } from '@/constants/unicode';
 import { useBalance } from '@/hooks';
+import { useReward } from '@/hooks/useReward';
 
 import { CardSection } from '../styled/CardSection';
 
+const { Text } = Typography;
 const Balance = styled.span`
   letter-spacing: -2px;
+  margin-right: 4px;
 `;
+const BalanceBreakdown = styled.div`
+  padding: 4px;
+`;
+const BalanceBreakdownLine = styled.div`
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  font-size: 14px;
+  color: #1f2229;
+
+  > span {
+    background: #fff;
+    z-index: 1;
+    &:first-child {
+      padding-right: 6px;
+    }
+    &:last-child {
+      padding-left: 6px;
+    }
+  }
+
+  &:before {
+    content: '';
+    position: absolute;
+    bottom: 6px;
+    width: 100%;
+    border-bottom: 2px dotted #c2cbd7;
+  }
+
+  &:not(:last-child) {
+    margin-bottom: 8px;
+  }
+`;
+const OVERLAY_STYLE = { maxWidth: '300px', width: '300px' };
+
+const CurrentBalance = () => {
+  const { totalOlasBalance, totalOlasStakedBalance } = useBalance();
+  const { accruedServiceStakingRewards } = useReward();
+
+  const balances = useMemo(() => {
+    return [
+      {
+        title: 'Staked amount',
+        value: balanceFormat(totalOlasStakedBalance ?? 0, 2),
+      },
+      {
+        title: 'Unclaimed rewards',
+        value: balanceFormat(accruedServiceStakingRewards ?? 0, 2),
```

Comment:
hope this is the correct value for "Unclaimed rewards"

Reply:
yeah it is, these are the rewards in contract that have not yet been claimed